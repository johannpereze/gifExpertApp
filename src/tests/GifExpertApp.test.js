import { shallow } from "enzyme"
import { GifExpertApp } from "../GifExpertApp"
import '@testing-library/jest-dom'

describe('Pruebas del componente GifExpertApp', () => {
    test('Debería hacer match con el snapshot', () => {
        const wrapper = shallow(<GifExpertApp />)

        expect(wrapper).toMatchSnapshot()
    })

    test('debe de mostrar una lista de categorías', () => {
        const categories = ["Evangelion", "Death Note"]
        const wrapper = shallow(<GifExpertApp defaultCategories={categories} />)
        expect(wrapper.find('GifGrid').length).toBe(categories.length)

    })


})
