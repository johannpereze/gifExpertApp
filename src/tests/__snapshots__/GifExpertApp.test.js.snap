// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente GifExpertApp Deber√≠a hacer match con el snapshot 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
